services/ext/api.go

```diff
@@ -294,13 +294,16 @@ func (api *PublicAPI) ActiveChats(parent context.Context) []*protocol.Chat {
 func (api *PublicAPI) DeleteChat(parent context.Context, chatID string) error {
 	return api.service.messenger.DeleteChat(chatID)
 }
+func (api *PublicAPI) MuteCommunityCategoryV2(request *requests.MuteCategory) error {
```

Comment:
A bit of documentation on this endpoint or on the the other `MuteCommunityCategory` could be useful to explain why v2 exists, or if the original endpoint is deprecated, etc.

I'm assuming we're duplicating because we don't want to break the mobile client? Or will Desktop need v1 and v2 simultaneously?

Reply:
@ilmotta the mobile never used it? :) if no - I will remove `V2`

Reply:
@borismelnik, let me check. I'll get back to this thread

Reply:
@borismelnik, I can confirm `MuteCommunityCategory` is not used by the mobile client.

Reply:
awesome! thanks.